extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -236,6 +219,47 @@ export function updateOverflow_(content, measurer, maxHeight, fontSize) {
   });
 }
 
+/**
+ *
+ * @param {!Document} document
+ * @param {!Element} element
+ */
+export function buildDom(document, element) {
+  const content = document.createElement('div');
+  applyFillContent(content);
+  content.classList.add(CONTENT_CLASS);
+
+  const contentWrapper = document.createElement('div');
+  contentWrapper.classList.add(CONTENT_WRAPPER_CLASS);
+  content.appendChild(contentWrapper);
+
+  const measurer = document.createElement('div');
+  measurer.classList.add(MEASURER_CLASS);
```

Comment:
Would there be value in `buildDom` being able to return a dict of elements to avoid the queries in `buildCallback`? Seems a shame that we have the references to the elements right here but still need multiple DOM queries to get handles on them.

Reply:
Deduping discussion to https://github.com/ampproject/amphtml/pull/35101#discussion_r663102371